tools/istio-iptables/pkg/cmd/run.go

```diff
@@ -520,8 +522,10 @@ func (cfg *IptablesConfigurator) run() {
 		cfg.iptables.InsertRule(iptableslog.UndefinedCommand, constants.ISTIOINBOUND, constants.MANGLE, 1,
 			"-p", constants.TCP, "-m", "mark", "--mark", cfg.cfg.InboundTProxyMark, "-j", constants.RETURN)
 		// prevent intercept traffic from envoy/pilot-agent ==> app by 127.0.0.6 --> podip
-		cfg.iptables.InsertRule(iptableslog.UndefinedCommand, constants.ISTIOINBOUND, constants.MANGLE, 2,
+		cfg.iptables.InsertRuleV4(iptableslog.UndefinedCommand, constants.ISTIOINBOUND, constants.MANGLE, 2,
```

Comment:
Any reasons not to use AppendVersionedRule ?

Reply:
we don't have InsertVersionedRule.

IMO we should remove all inserts in the future but currently we don't have it

Reply:
Got it. +1 on killing the insert!